TabClasses/SlowADCs.py

```diff
@@ -0,0 +1,35 @@
+from functools import partial
+from slsdet import Detector, dacIndex, readoutMode, runStatus
+
+
+class SlowADCs:
+    def __init__(self, mainWindow):
+        self.mainWindow = mainWindow
+
+    def setup_ui(self):
```

Comment:
I guess this function is there to be consistent with the other tabs? Or is it required?

Reply:
It is not required. As you have said it is there just to be consistent. 
Maybe in a future change we iterate over tabs and use `tab.setup_ui()`. This way we won't get errors.